### ISSUE TITLE
Flutter plugin doesn't detect existing Flutter sdk

### ISSUE THREAD

**Issue by DaveShuckerow** [flutter/flutter-intellij#1199]
## Steps to Reproduce

I have Flutter installed and am setting up the Flutter plugin in IntelliJ CE 2017.1.

The plugin should be able to detect where Flutter is installed, but does not:

![image](https://user-images.githubusercontent.com/14947366/28087835-b3ee2240-6638-11e7-9776-9d4ef54b5a53.png)

## Version info

$ flutter doctor
[✓] Flutter (on Linux, locale en_US.UTF-8, channel master)
    • Flutter at ...djshuckerow/Code/flutter
    • Framework revision 792c98751f (19 hours ago), 2017-07-10 17:37:21 -0700
    • Engine revision 8ad1576f27
    • Tools Dart version 1.25.0-dev.4.0

[✓] Android toolchain - develop for Android devices (Android SDK 25.0.3)
    • Android SDK at ...
    • Platform android-stable, build-tools 25.0.3
    • ANDROID_HOME = ...
    • Java binary at: ...
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] Android Studio (version 2.3)
    • Android Studio at ...
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] Android Studio (version 2.2)
    • Android Studio at ...
    • Java version OpenJDK Runtime Environment (build 1.8.0_76-release-b03)

[✓] IntelliJ IDEA Community Edition (version 2017.1)
    • Flutter plugin version 15.1
    • Dart plugin version 171.4424

[✓] Connected devices
    • None

**Comment by pq** [comment]
Absolutely.  We've talked a bit about this and it's very much on deck.  Thanks for the report!

**Comment by jwren** [comment]
If the Flutter SDK is not set, `which flutter` should be used to get and set the Flutter and Dart SDK settings.

**Comment by jwren** [comment]
I went to add this, but it looks like support was added in https://github.com/flutter/flutter-intellij/commit/50839574ea83b70c6e53c011851e14a7eb427b0f, and this issue was just never closed out.
